apps/judicial-system/backend/src/app/modules/defendant/defendant.service.ts

```diff
@@ -100,4 +102,31 @@ export class DefendantService {
 
     return true
   }
+
+  async isDefendantInActiveCustody(defendants?: Defendant[]): Promise<boolean> {
+    if (
+      !defendants ||
+      !defendants[0]?.nationalId ||
+      defendants[0]?.noNationalId
+    ) {
+      return false
+    }
+
+    const defendantsInCustody = await this.defendantModel.findAll({
+      include: [
+        {
+          model: Case,
+          as: 'case',
+          where: {
+            state: CaseState.ACCEPTED,
+            type: CaseType.CUSTODY,
+            valid_to_date: { [Op.gte]: literal('current_date') },
+          },
+        },
+      ],
+      where: { nationalId: defendants[0].nationalId },
+    })
+
+    return defendantsInCustody.some((d) => d.case)
```

Comment:
Why do you use `.some` here? You would always get a single case from the query above, since a defendant can only serve a single custody at a time, so wouldn't it make sense to do 

`return defendantsInCustody[0].case` 

instead?

Reply:
You are right, we will always get a single defendant from the result in production. I want to keep it like this since our system can represent this state (a defendant is in multiple active custody at once). It is likely to happen in dev and staging env.